diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7a5453a..e2b10e7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -28,10 +28,10 @@ if(COMMAND cmake_policy)
     endif(POLICY CMP0072)
 endif(COMMAND cmake_policy)
 
-find_program(CCACHE_PROGRAM ccache)  #This check should occur before project()
-if(CCACHE_PROGRAM)
-    set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE "${CCACHE_PROGRAM}")
-endif()
+#find_program(CCACHE_PROGRAM ccache)  #This check should occur before project()
+#if(CCACHE_PROGRAM)
+#    set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE "${CCACHE_PROGRAM}")
+#endif()
 
 # include local  modules
 include(AddFileDependencies)
