--- setup.py.orig	2015-11-24 14:25:36.219862475 +0000
+++ setup.py	2015-11-24 14:50:00.704828553 +0000
@@ -319,19 +319,11 @@
     packages = ['Onboard', 'Onboard.pypredict'],
 
     data_files = [('share/glib-2.0/schemas', glob.glob('data/*.gschema.xml')),
-                  ('share/onboard', glob.glob('AUTHORS')),
-                  ('share/onboard', glob.glob('CHANGELOG')),
-                  ('share/onboard', glob.glob('COPYING')),
-                  ('share/onboard', glob.glob('NEWS')),
-                  ('share/onboard', glob.glob('README')),
-                  ('share/onboard', glob.glob('onboard-defaults.conf.example')),
-                  ('share/onboard', glob.glob('onboard-defaults.conf.example.nexus7')),
                   ('share/icons/hicolor/scalable/apps', glob.glob('icons/hicolor/*')),
                   ('share/icons/HighContrast/scalable/apps', glob.glob('icons/HighContrast/*')),
                   ('share/icons/ubuntu-mono-dark/status/22', glob.glob('icons/ubuntu-mono-dark/*')),
                   ('share/icons/ubuntu-mono-light/status/22', glob.glob('icons/ubuntu-mono-light/*')),
                   ('share/sounds/freedesktop/stereo', glob.glob('sounds/*')),
-                  ('share/onboard/docs', glob.glob('docs/*')),
                   ('share/onboard/layouts', glob.glob('layouts/*.*')),
                   ('share/onboard/layouts/images', glob.glob('layouts/images/*')),
                   ('share/onboard/themes', glob.glob('themes/*')),
